consensus/raft/raft.go

```diff
@@ -79,6 +82,7 @@ func newRaftWrapper(
 	raftW := &raftWrapper{}
 	raftW.config = cfg
 	raftW.host = host
+	raftW.peerManager = pstoremgr.New(context.Background(), host, "")
```

Comment:
unsure we need to carry this, if you just need a `host.Peerstore().ClearAddrs(pid)` inside the `observePeers()` function.